web/packages/teleterm/src/ui/uri.ts

```diff
@@ -62,6 +64,9 @@ export type ResourceUri = RootClusterResourceUri | LeafClusterResourceUri;
 export type ClusterUri = RootClusterUri | LeafClusterUri;
 export type ServerUri = RootClusterServerUri | LeafClusterServerUri;
 export type KubeUri = RootClusterKubeUri | LeafClusterKubeUri;
+export type KubeResourceUri =
+  | RootClusterKubeResourceUri
+  | LeafClusterKubeResourceUri;
```

Comment:
It seems to me that we should rename these types to `*ResourceNamespaceUri` for consistency.